nwcsafpps_runner/publish_and_listen.py

```diff
@@ -51,7 +51,7 @@ def run(self):
         LOG.debug("Subscribe topics = %s", str(self.subscribe_topics))
         with posttroll.subscriber.Subscribe("", self.subscribe_topics, True) as subscr:
 
-            for msg in subscr.recv(timeout=90):
+            for msg in subscr.recv():
```

Comment:
Not sure why we had a timeout of 90 seconds before, but is it safe to change this? Why was this needed?

Reply:
I removed it because the patched subscriber did not have the timeout keyword. I am not sure if it is safe to remove.

Reply:
The timeout is needed I think when the thread is shut down. Maybe something we need in the patched_subscriber then.